examples/create-check.js

```diff
@@ -20,49 +20,52 @@ Lob.addresses.create({
   address_country: 'US',
 }, function (err, address) {
   if (err) {
-    console.log(err);
-  } else {
-    Lob.bankAccounts.create({
-      routing_number: '122100024',
-      account_number: '123456789',
-      bank_address: {
-        name: 'Chase Bank',
-        address_line1: '55 Edmonds',
-        address_city: 'Palo Alto',
-        address_state: 'CA',
-        address_zip: '90081',
-        address_country: 'US'
-      },
-      account_address: {
-        name: 'Leore Avidar',
-        address_line1: '123 Test Street',
-        address_city: 'Sunnyvale',
-        address_state: 'CA',
-        address_zip: '94085',
-        address_country: 'US'
-      },
-      signatory: 'John Doe'
-    }, function (err, bankAccount) {
+    return console.log(err);
+  }
+  Lob.bankAccounts.create({
+    routing_number: '122100024',
+    account_number: '123456789',
+    bank_address: {
+      name: 'Chase Bank',
+      address_line1: '55 Edmonds',
+      address_city: 'Palo Alto',
+      address_state: 'CA',
+      address_zip: '90081',
+      address_country: 'US'
+    },
+    account_address: {
+      name: 'Leore Avidar',
+      address_line1: '123 Test Street',
+      address_city: 'Sunnyvale',
+      address_state: 'CA',
+      address_zip: '94085',
+      address_country: 'US'
+    },
+    signatory: 'John Doe'
+  }, function (err, bankAccount) {
+    if (err) {
+      return console.log(err);
+    }
+    Lob.bankAccounts.verify(bankAccount.id, [23, 34], function(err) {
```

Comment:
It actually looks like that check isn't in the linter cause literally all of them don't have spaces. I didn't add a space here for consistency's sake. Should make an issue to add this to the linter?